docs/source/usage/particles.rst

```diff
@@ -38,74 +40,76 @@ FillAllGaps
 .. doxygenstruct:: picongpu::particles::FillAllGaps
    :project: PIConGPU
 
+Manipulation Filters
+--------------------
 
-Manipulation
-------------
+IsHandleValid
```

Comment:
if this is the default filter for all functors the user faces, state that here in a sentence